linkerd/proxy/server-policy/Cargo.toml

```
@@ -10,6 +10,7 @@ publish = false
 proto = ["linkerd-http-route/proto", "linkerd2-proxy-api", "prost-types"]
 
 [dependencies]
+governor = { version = "0.6", default-features = false, features = ["std", "jitter", "quanta"] }
```

```suggestion
governor = { version = "0.6", default-features = false, features = ["std", "jitter"] }
```

We're not using the quanta clock so it's fine to omit it.